### PR TITLE
Gradle update, project cleanup

### PR DESCRIPTION
**Pull request by rosolko**
* Remove local idea configs
* Update gradle scripts and version to 3.5
* Fix locators
* Fix tests on travis ci

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
```
Comment (asolntsev) [inline .editorconfig]: I typically use 2, not 4.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,6 @@
 build
 hs_err_pid*
-.gradle
\ No newline at end of file
+.gradle
+.idea
+!gradle-wrapper.jar
```
Comment (asolntsev) [inline .gitignore]: why ignore gradle-wrapper.jar? Typically people commit it to git.

File: .idea/modules.xml [hunk 3]
```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (asolntsev) [inline .idea/modules.xml]: why remove idea files? I need them.

File: .gitignore (hunk 2 above)
Comment (rosolko) [inline .gitignore]: Usually people generate this file by syncing gradle or execute wrapper task.

File: .editorconfig (hunk 1 above)
Comment (rosolko) [inline .editorconfig]: Ok. I will change it

File: .idea/modules.xml (hunk 3 above)
Comment (rosolko) [inline .idea/modules.xml]: As I know it's user specific thing. And it is generating while you import gradle project to your IDE. Am I right?

**Comment by asolntsev** [comment]
@rosolko Great! Thank you. PR merged.

**Pull request merged**